### DISCUSSION TITLE
Coverage reporting with --cov-report annotate will be going away

### DISCUSSION THREAD

**Discussion by adeak** [pyvista/pyvista#1994] [Show and tell]
This is just an FYI because I only learned about `--cov-report annotate` thanks to PyVista.

[`coveragepy` has deprecated this method](https://github.com/nedbat/coveragepy/commit/59b07a12de95de776fa2f44a2101f63a156fee75) in favour of "more modern reporting tools, including the `html_report()` method". I haven't checked that out yet, but if any of you are relying on it too, you should know about the deprecation.

(We don't mention this method anywhere in the contributing guide, I checked.)

**Comment by akaszynski** [comment]
It's too bad. I prefer this method as I can use my own text editor.

Thanks for the heads up!
